index.html

```diff
@@ -609,6 +609,77 @@ <h3>Ed25519Signature2018</h3>
           </pre>
       </section>
 
+      <section id="JsonWebSignature2020">
```

Comment:
Most ids in this doc have microdata.  I assume this was copied from the above which doesn't have that info.

Reply:
would you mind commenting on this issue so I can address all these cases at once: https://github.com/w3c-ccg/security-vocab/issues/49

Reply:
Not sure what to comment on.  The title seems good enough.

Reply:
leave advice on the issue so that someone less familar with microdata can take a stab a resolving it. I don't know how to address your concern based on your comments here.